gr-uhd/lib/rfnoc_rx_streamer_impl.cc

```diff
@@ -58,7 +67,9 @@ rfnoc_rx_streamer_impl::rfnoc_rx_streamer_impl(rfnoc_graph::sptr graph,
       d_streamer(graph->create_rx_streamer(num_chans, stream_args)),
       d_unique_id(
           std::dynamic_pointer_cast<::uhd::rfnoc::node_t>(d_streamer)->get_unique_id()),
-      d_issue_stream_cmd_on_start(issue_stream_cmd_on_start)
+      d_issue_stream_cmd_on_start(issue_stream_cmd_on_start),
+      d_start_time_set(start_time_set),
+      d_start_time(start_time)
```

Comment:
Where do these get used? I'm probably staring right at it, but I can't figure it out...

Reply:
These are used in the `start()` method, as you pointed out in your comment above:
https://github.com/mbr0wn/gnuradio/blob/70ec6a0877f01b248babe437044b7c90bdf4833c/gr-uhd/lib/rfnoc_rx_streamer_impl.cc#L82-L90

I did not modify the method, just added the parameters to the constructor, so they can be set from within GNU Radio. What modifications to `start()` do you have in mind?